packages/injected/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/injected-wallets",
-  "version": "2.8.4",
+  "version": "2.9.0-alpha.1",
```

```suggestion
  "version": "2.8.5-alpha.1",
```
As we are only adding wallet support and not changing functionality we will keep the version change at a patch